lighthouse-core/fraggle-rock/user-flow.js

```diff
@@ -174,13 +182,19 @@ class UserFlow {
   async startTimespan(stepOptions) {
     if (this.currentTimespan) throw new Error('Timespan already in progress');
     if (this.currentNavigation) throw new Error('Navigation already in progress');
+    const options = {...this._options, ...stepOptions};
+
+    if (this._dryRun) {
+      await dryRun('timespan', options);
```

Comment:
What happens to emulation when the driver disconnects? Does it persist or go away?